value_converter_test.go

```diff
@@ -0,0 +1,31 @@
+package clickhouse
+
+import (
+	"database/sql/driver"
+	"reflect"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestConverter(t *testing.T) {
+	testCases := []struct {
+		value    interface{}
+		expected driver.Value
+		msg      string
+	}{
+		{int64(maxAllowedUInt64), int64(9223372036854775807), "int64(maxAllowedUInt64)"},
+		{uint64(maxAllowedUInt64 + 1), []byte("9223372036854775808"), "uint64(maxAllowedUInt64+1)"},
+		{uint64(maxAllowedUInt64*2 + 1), []byte("18446744073709551615"), "uint64(maxUInt64)"},
+	}
```

Comment:
Good, but need also check that other values (for the instance float, string, driver.Value, pointer do handled properly)
And it would be great to check that converter works when sql.DB is used.
```
type ConverterSuite struct {
	chSuite
}

func (s * ConverterSuite) TestBigInt() {
       result, err := s.conn.Exec("INSERT INTO data (u64) VALUES (?)", maxAllowedUInt64 + 1)
       s.NoError(err)
       rows, err := s.conn.Query("SELECT u64 FROM data WHERE u64=?", maxAllowedUInt64 + 1)
       s.NoError(err)
       v, err := scanValues(rows, tc.args)
       s.NoError(err) {
       s.Equal([][]interface{}{maxAllowedUInt64 + 1}, v)
       s.NoError(rows.Close())
       // the same for a statements
}

func TestConveter(t *testing.T) {
	suite.Run(t, new(ConverterSuite))
}
```

Reply:
Sure, but it do not corresponds whith uint64 high bit issue. Probably the better case is to merge fix actual code changes and then we extend tests

Reply:
ок